coalib/results/Diff.py

```diff
@@ -438,9 +436,15 @@ def change_line(self, line_nr, original_line, replacement):
                                                replacement)
             replacement = ''.join((orig_diff + new_diff).modified)
 
-        linediff.change = (original_line, replacement)
+        linediff.change = (self._file[line_nr-1], replacement)
         self._changes[line_nr] = linediff
 
+    def change_line(self, line_nr, original_line, replacement):
+        # Use of change_line method is deprecated
```

Comment:
sorry I really wanted to merge this but *this comment is useless* it doesn't tell me anything that sholdn't be in the warning